### PR TITLE
add example to the doc of combine operator

### PR DESCRIPTION
**Pull request by muskan3006**
The combine operator doc was missing an example, added it.

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

**Review by johanandren — APPROVED** [review]
LGTM, makes sense to re-use those pre-existing sample snippets.

**Comment by muskan3006** [comment]
Hey @octonato @ignasi35 can you please review it

**Review by octonato — APPROVED** [review]
Thanks!

File: akka-docs/src/main/paradox/stream/operators/Sink/combine.md [hunk 1]
```diff
@@ -6,12 +6,29 @@ Combine several sinks into one using a user specified strategy
 
 ## Signature
 
-@apidoc[Sink.combine](Sink$) { scala="#combine[T,U](first:akka.stream.scaladsl.Sink[U,_],second:akka.stream.scaladsl.Sink[U,_],rest:akka.stream.scaladsl.Sink[U,_]*)(strategy:Int=&gt;akka.stream.Graph[akka.stream.UniformFanOutShape[T,U],akka.NotUsed]):akka.stream.scaladsl.Sink[T,akka.NotUsed]" java="#combine(akka.stream.javadsl.Sink,akka.stream.javadsl.Sink,java.util.List,akka.japi.function.Function)" }
+@apidoc[Sink.combine](Sink$) { scala="
```
Comment (leviramsey) [inline akka-docs/src/main/paradox/stream/operators/Sink/combine.md:9]: I think the wrapping here messed up the doc that got published

**Pull request merged**